roles/common/tasks/main.yml

```diff
@@ -7,6 +7,8 @@
     - empty.yml
   tags:
     - vars
+    # We need these vars for the entitlements tag to work
+    - entitlements
```

Comment:
I think if we want to switch from using the vars tag to the always tag, we should do that in a separate PR.